src/App.js

```diff
@@ -1,16 +1,90 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm'
+
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
-  render() {
-    return (
-      <div>
-        <h2>Welcome to your Todo App!</h2>
-      </div>
-    );
+
+  constructor(props) {
+  super();
+  this.state ={
+    currentList: [
+      {
+      task: 'Organize Garage',
+      id: 1528817077286,
+      completed: false,
+      class: ''
+      },
+
+      
+
+    {
+      task:'Bake Cookies',
+      id: 1528817084358,
+      completed: false
+      
+    }
+  ],
+  inputValue: ''
+
+}
   }
+
+  handleChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+
+  });
+
+}
+
+
+addItem = e => {
+  e.preventDefault();
```

Comment:
Great use of the updated arrow function method syntax and smart application of e.preventDefault() here.